gulpfile.js

```diff
@@ -47,17 +47,6 @@ async function clean() {
     return fs.emptyDir(distdir);
 }
 
-function setTelemetryTarget() {
```

Comment:
Please bring these files back as this pattern of using different files that are code is consistent with how we manage this stuff in our other NodeJS products. The fact that they're still Dev vs Prod configuration still applies in the new *-Preview deployment model, it's just that the Preview ext will currently always pick the Dev configuration.
In the near future when we try to support EUDB regions, I only want to modify these code files, and they'll reuse code we've creted elsewhere so we don't reinvent the wheel.

Reply:
The point of setting these _at package time_ was to prevent the need to build twice.  Looking at the official builds though, the build+package step is only 45 seconds, so it should be fine to go the build-twice route.

Reply:
Switched to packaging a single VSIX in the package step, and changed the Official Build to build twice to generate both packages